src/Notify/NotifyMesssage.js

```diff
@@ -54,7 +61,7 @@ export default class NotifyMesssage extends Component {
         style={{ overflow: 'hidden', transition: 'margin ease-out 300ms', ...styles[this.state.isOpen ? 1 : 0] }}
       >
         <a className="uk-close" onClick={this.handleClose} />
-        <div>
+        <div onClick={this.handleClick}>
```

Comment:
pass the prop directly here instead of wrapping it in handleClick since we're not doing anything special